src/commands/System/eval.ts

```diff
@@ -1,67 +1,35 @@
-import { EvlnCommand, EvlynCommandOptions } from '@lib/structures/SkyraCommand';
-import { Events, PermissionLevels } from '@lib/types/Enums';
+import { Stopwatch } from '@klasa/stopwatch';
```

Comment:
Why nuke so much in this file? I mean I know it's just Evlyn but still.

Reply:
There's no flag support in Sapphire yet, and Evlyn isn't a bot where we'd try... very complicated stuff.